### PR TITLE
nicer async dynamic

### PR DESCRIPTION
**Pull request by JosuaKrause**
getting response is blocking

### CODE REVIEW

File: accern_xyme/accern_xyme.py [hunk 1]
```diff
@@ -1470,19 +1468,21 @@ def __init__(self, pipeline: PipelineHandle, data_id: str) -> None:
         self._data_id = data_id
         self._value: Optional[ByteResponse] = None
 
-    def maybe_value(self) -> Optional[ByteResponse]:
+    def has_fetched(self) -> bool:
+        return self._value is not None
+
+    def get(self) -> Optional[ByteResponse]:
         if self._value is None:
```
Comment (jckhang) [inline accern_xyme/accern_xyme.py:1475]: if not self.has_fetched()

Comment (JosuaKrause) [reply accern_xyme/accern_xyme.py:1475]: sure

Comment (JosuaKrause) [reply accern_xyme/accern_xyme.py:1475]: no we need that for the type checker to know it is not None

Comment (jckhang) [reply accern_xyme/accern_xyme.py:1475]: okay

**Pull request merged**